src/main/java/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -189,18 +190,21 @@
     return Match.anyOf(Matches.territoryHasNoEnemyUnits(player, data), Matches.territoryIsInList(clearedTerritories));
   }
 
-  public static Predicate<Territory> territoryIsEnemyOrHasEnemyUnitsOrCantBeHeld(final PlayerID player, final GameData data,
+  public static Predicate<Territory> territoryIsEnemyOrHasEnemyUnitsOrCantBeHeld(final PlayerID player,
+      final GameData data,
       final List<Territory> territoriesThatCantBeHeld) {
     return Match.anyOf(Matches.isTerritoryEnemyAndNotUnownedWater(player, data),
         Matches.territoryHasEnemyUnits(player, data), Matches.territoryIsInList(territoriesThatCantBeHeld));
   }
 
   public static Predicate<Territory> territoryHasInfraFactoryAndIsLand() {
-    final Predicate<Unit> infraFactoryMatch = Match.allOf(Matches.unitCanProduceUnits(), Matches.unitIsInfrastructure());
+    final Predicate<Unit> infraFactoryMatch =
+        Match.allOf(Matches.unitCanProduceUnits(), Matches.unitIsInfrastructure());
     return Match.allOf(Matches.territoryIsLand(), Matches.territoryHasUnitsThatMatch(infraFactoryMatch));
   }
```

Comment:
This method now requires a Javadoc because it has more than two lines due to wrapping.  You can probably just remove the `Match` suffix from the local variable and re-wrap the line.